launcher/src/test/java/org/apache/spark/launcher/SparkSubmitCommandBuilderSuite.java

```diff
@@ -250,6 +250,34 @@ public void testMissingAppResource() {
     new SparkSubmitCommandBuilder().buildSparkSubmitArgs();
   }
 
+  @Test
+  public void testIsClientMode() {
+    // Default master is "local[*]"
+    SparkSubmitCommandBuilder launcher =
+            newCommandBuilder(Collections.emptyList());
+    assertTrue("By default application run in local mode",
+            launcher.isClientMode(Collections.EMPTY_MAP));
```

Comment:
Indentation?  We use 2-space indentation in Java file. Please see the other test case.